### PR TITLE
SK 

### PR DESCRIPTION
**Pull request by Steven-Klavins**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+@font-face {
+src: url(fonts/MISTV___.TTF); 
+font-family: custom-font;  
+}
+
+@font-face {
+    src: url(fonts/desyrel.ttf); 
+    font-family: pen;  
+    }
+
```
Comment (AlexLewis10) [inline index.css:10]: YAY A NICE UI

File: jasmine-standalone-3.5.0/spec/tenPinAppSpec.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+describe('Bowling', function() {
+    var bowling; 
+
+    beforeEach(function() {
+        bowling = new Bowling()
+      });
+
```
Comment (AlexLewis10) [inline jasmine-standalone-3.5.0/spec/tenPinAppSpec.js:7]: Good use of beforeEach for some DRY code

File: jasmine-standalone-3.5.0/spec/tenPinAppSpec.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        
+    })
+
+    it ("checks to see if the player gets a bonus roll at the end of the game", function(){
+        bowling.frame = 10;
+        bowling.calcPoints (10,0)
+        expect (bowling.bonus).toEqual(true)
+    })
+
+
```
Comment (AlexLewis10) [inline jasmine-standalone-3.5.0/spec/tenPinAppSpec.js:60]: It would have been nice to see more tests. There are loads of edge cases in this

File: jasmine-standalone-3.5.0/src/tenPinApp.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+function Bowling () {
+this.player_score = 0;
+this.strike = false;
+this.spare = false;
+this.frame = 0;
+this.bonus = false;
+}
+
```
Comment (AlexLewis10) [inline jasmine-standalone-3.5.0/src/tenPinApp.js:8]: Method names clearly described

File: jasmine-standalone-3.5.0/src/tenPinController.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+                if (prv == 10){
+                     bowling.calcPoints(10,0)
+                     $("#the").text(bowling.player_score); 
+                     $(arr[elm]).replaceWith(function() {
+                    return '<div>' + bowling.player_score +  '</div>';  
+            })
+           
+            elm++  
+            two = false
```
Comment (AlexLewis10) [inline jasmine-standalone-3.5.0/src/tenPinController.js:26]: I think elm is a bit of a dodgy name here, could do with something being clearer

File: jasmine-standalone-3.5.0/src/tenPinController.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
+
+
```
Comment (AlexLewis10) [inline jasmine-standalone-3.5.0/src/tenPinController.js:2]: Personally, I'm finding this controller a little confusing, it isn't obvious what elm, prv or two are

File: jasmine-standalone-3.5.0/spec/tenPinAppSpec.js [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+        bowling.calcPoints (2,5)
+        expect (bowling.player_score).toEqual(24)
+    })
+
+    it ("doesn't add bonus following round if player scored a strike twice", function(){
+        bowling.calcPoints (10,0)
+        bowling.calcPoints (10,0)
+        expect (bowling.player_score).toEqual(20)
+    })
+
```
Comment (AlexLewis10) [inline jasmine-standalone-3.5.0/spec/tenPinAppSpec.js:33]: style and indentation looks consistent throughout

File: jasmine-standalone-3.5.0/spec/tenPinAppSpec.js [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+    it ( "Has has a starting score of 0 for player", function () {
+        expect (bowling.player_score).toEqual(0)
+    }) 
+
+    it ( "Can add points to a players score", function () {
+        bowling.calcPoints(3,0)
+        expect (bowling.player_score).toEqual(3)
+    }) 
+
```
Comment (AlexLewis10) [inline jasmine-standalone-3.5.0/spec/tenPinAppSpec.js:16]: Could you have added some describe blocks to this?

File: index.css [hunk 9]
```diff
@@ -0,0 +1,90 @@
+@font-face {
```
Comment (AlexLewis10) [inline index.css:1]: There is no readme :-(

File: jasmine-standalone-3.5.0/src/tenPinApp.js [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+Bowling.prototype.calcPoints = function (r1,r2){
```
Comment (AlexLewis10) [inline jasmine-standalone-3.5.0/src/tenPinApp.js:9]: Did you plan model/beforehand to make sure that you understand the rules correctly? Did you go and look at other examples?

File: jasmine-standalone-3.5.0/src/tenPinApp.js [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+        this.player_score +=10;   
+    }
+     else if (this.strike == true){
+        this.player_score += (r1 + r2) * 2;
+        this.strike = false;
+    }
+    else if (this.spare == true){
+        this.spare = false;
+        this.player_score += (r1*2)+r2
+    }
```
Comment (AlexLewis10) [inline jasmine-standalone-3.5.0/src/tenPinApp.js:22]: this method is a little bit long. You have the conditionals and then doing 2 extra things within each conditional. Does this follow SRP?

File: jasmine-standalone-3.5.0/src/tenPinApp.js (hunk 10 above)
Comment (alicelieutier) [reply jasmine-standalone-3.5.0/src/tenPinApp.js:9]: It feels like this should be called differently. Maybe addRoll or something.

File: jasmine-standalone-3.5.0/src/tenPinApp.js (hunk 11 above)
Comment (alicelieutier) [reply jasmine-standalone-3.5.0/src/tenPinApp.js:22]: Just because it's long doesn't mean it doesn't follow SRP. SRP means doing one thing, but it can be a complex thing, as long as it is serving one purpose (here adding up the score).

File: jasmine-standalone-3.5.0/src/tenPinController.js (hunk 6 above)
Comment (alicelieutier) [reply jasmine-standalone-3.5.0/src/tenPinController.js:2]: Agreed - use full, meaningful names
